runtime/include/cloe/data_broker.hpp

```diff
@@ -810,17 +794,16 @@ class DataBroker {
     * \param new_name New name of the signal
     * \return Pointer to the signal
     */
-  signal_ptr_t alias(signal_ptr_t signal, const char* new_name) {
-    std::string name = new_name;
-    if (name.empty()) {
+  signal_ptr_t alias(signal_ptr_t signal, std::string_view new_name) {
+    if (new_name.empty()) {
       throw std::invalid_argument(
           signal->names().size()
               ? fmt::format("The alias name for signal '{}' must not be an empty string",
                             signal->name())
               : fmt::format("The name for a signal must not be an empty string"));
     }
     std::pair<signal_container_t::iterator, bool> inserted =
-        signals_.try_emplace(new_name, std::move(signal));
+        signals_.try_emplace((std::string)new_name, std::move(signal));

```

Comment:
instead of the c-style cast you can also call `static_cast<std::string>(stringViewInstance)`, this is guaranteed to call the string ctor and create a brand-new null-terminated string instance that can be emplaced

Reply:
Done.